bindings/http/http.go

```diff
@@ -186,6 +196,7 @@ func (h *HTTPSource) Operations() []bindings.OperationKind {
 func (h *HTTPSource) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
 	u := h.metadata.URL
 
+	traceHeaders := h.traceHeaders
```

Comment:
No need for this, just access h.traceHeader

Reply:
I’ll remove. I was trying to match current code, thought maybe it was convention. :)